frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx

```diff
@@ -21,16 +19,12 @@ const RobotView = styled.div`
     gap: 1rem;
 `
 
-const isRobotBlocked = (robot: Robot): boolean => {
-    return robot.status === 'Blocked'
-}
-
 export const RobotStatusSection = () => {
     const { TranslateText } = useLanguageContext()
     const { installationCode } = useInstallationContext()
     const { enabledRobots } = useRobotContext()
     const { switchSafeZoneStatus } = useSafeZoneContext()
-    const { setAlert } = useAlertContext()
+
```

Comment:
Is this an extra un-necessary blank line ?

Reply:
Its intentional to create some more air to increase readability